docs/ui/layouts/Column/index.tsx

```diff
@@ -0,0 +1,77 @@
+import { css } from '@emotion/react';
+import { breakpoints, theme } from '@expo/styleguide';
+import React, { PropsWithChildren, ReactNode } from 'react';
+
+import { Scroll } from './Scroll';
+
+export { Scroll } from './Scroll';
+
+type ColumnLayoutProps = PropsWithChildren<{
+  /** The navigation sidebar content, either in the right sidebar or overlayed in mobile */
+  navigation: ReactNode;
+  /** The optional table of contents sidebar, adds padding to main content when omitted */
+  sidebar?: ReactNode;
+}>;
+
+export function ColumnLayout({ navigation, sidebar, children }: ColumnLayoutProps) {
+  return (
+    <div css={containerStyle}>
+      <div css={navigationColumnStyle}>{navigation}</div>
+      <div css={articleColumnStyle}>
+        <Scroll css={!sidebar && paddedArticleColumnStyle}>
+          <div css={contentBehaviorStyle}>{children}</div>
+        </Scroll>
```

Comment:
Wrapping the stuff like this keeps the content scrollbar on the outer right of the content. We can simplify this too by moving the behavior to Scroll, but it will keep the content scrollbar "decoupled" for the right sidebar